virt/virt-2-4-release-notes.adoc

```diff
@@ -68,10 +117,6 @@ obsolete IP address. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1686208[*
 $ oc get pod -o wide
 ----
 
-* Some resources are improperly retained when removing {VirtProductName}. You
```

Comment:
Just making sure that this was removed on purpose (note above says this section is unchanged)

Reply:
I would like to add this content. Could you please provide the entire text. I tried looking into the 2.3 rel notes https://docs.openshift.com/container-platform/4.4/cnv/cnv_release_notes/cnv-2-3-release-notes.html#cnv-2-3-known-issues to see if I could grab the text from there, but I couldn't find it there.

Reply:
@lmandavi Yeah, I can confirm it's not in the file. Which reminds me there was some strangeness regarding this file when the virt namechange was made - grabbed the 2.2 release notes file or something? Either way, if it's not in 2.3 RN, it doesn't matter for us now :)